docs/guides/getting-started-java-spring.md

```diff
@@ -0,0 +1,256 @@
+3. Add a start event, and name it `Payment request received`.
+4. Add a task named `Prepare transaction`. Click the wrench-shaped icon to change the type of task to a script task, and configure the following properties:
+   1. **Implementation**: `FEEL expression`
+   2. **Script/Result variable**: `totalWithTax`
+   3. **Script/FEEL expression**: `total * 1.1` (this represents the tax applied to the transaction.)
+5. Add a task named `Charge credit card`.
+6. Click on the task and click the wrench-shaped icon to change the type of task to a service task. In the properties panel, change the **Task definition/Type** to `charge-credit-card`.
+7. Add an end event named `Payment executed`.
+8. Save this BPMN file to your Spring project in `src/main/resources`, and name it `process-payments.bpmn`.
+
+## Step 4: Deploy your process
+
+To deploy your process, take the following steps:
+
+1. Open Desktop Modeler and click the rocket icon in the bottom left corner.
+2. Change the **Deployment name** to `process-payments`, and ensure the **Target** is `Camunda 8 Self-Managed`.
+3. Change the **Cluster endpoint** to `http://localhost:26500/`, with no authentication.
+4. Click **Deploy**.
+
+When you open Operate, you should now note the process deployed to your local Self-
+Managed setup.
+
+## Step 5: Run your process from Modeler
+
+To run your process, take the following steps:
+
+1. From Desktop Modeler, click the "play" icon (next to the rocket icon to deploy) in the bottom left corner.
+2. In **Variables**, insert the JSON object `{"total": 100}`.
+3. Click **Start**.
+
+From Operate, you should now notice a process instance running. You'll notice the process has stopped at **Charge credit card**, because we'll need to configure a job worker.
+
+## Step 6: Implement a service task
+
+To implement a service task, take the following steps:
+
+### Configure Spring Boot Starter
+
+See our documentation on [adding the Spring Zeebe SDK to your project](/apis-tools/spring-zeebe-sdk/getting-started.md#add-the-spring-zeebe-sdk-to-your-project) for more details, also described below:
+
+1. Copy the following code snippet into the `pom.xml` file of your Spring project, below properties and above dependencies:
+
+```
+<repositories>
+   <repository>
+      <releases>
+         <enabled>true</enabled>
+      </releases>
+      <snapshots>
+         <enabled>false</enabled>
+      </snapshots>
+      <id>identity</id>
+      <name>Camunda Identity</name>
+      <url>https://artifacts.camunda.com/artifactory/camunda-identity/</url>
+   </repository>
+</repositories>
+```
+
+2. Add the following dependency to your `pom.xml` file:
+
+```
+<dependency>
+   <groupId>io.camunda</groupId>
+   <artifactId>spring-boot-starter-camunda-sdk</artifactId>
+   <version>8.5.0</version>
+</dependency>
+```
+
+### Configure the Zeebe client
+
+Open your `src/main/resources/application.properties` file, and paste the following snippet to connect to a Self-Managed Zeebe broker:
```

Comment:
Nit pick, but I think we can say "the" here, as this setup only has one Zeebe broker.